app/messaging/schemas/payment-schedule.js

```diff
@@ -4,5 +4,6 @@ module.exports = Joi.object({
   order: Joi.number().required(),
   dueDate: Joi.string().optional().allow('', null),
   period: Joi.string().optional().allow('', null),
-  value: Joi.string().required()
+  value: Joi.string().required(),
+  paymentType: Joi.string().required()
```

Comment:
can only be 1 of 2 values: "Immediate payment" and "Quarterly payment"

Reply:
needs added to the `asyncapi`

Reply:
Asyncapi review ticket already created